vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -1 +1,107 @@
 # How to use figures
+
+This guide shows you how to add any dash component to Vizro that needs to be reactive to controls or actions in your page.
+If you want to add a static Dash component to your page, use [custom components](custom-components.md) instead.
+
+In essence, `Figure` is a higher-level abstraction, providing a flexible foundation for all types of reactive Dash
+components in Vizro. The [`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table] and [`AgGrid`][vizro.models.AgGrid]
+are specific implementations of the `Figure`. They serve as intuitive shortcuts, embedding behaviors and
+interactions specific to their purposes.
+
+The following flowchart shows what you need to consider when choosing which model to use:
+
+``` mermaid
```

Comment:
We had some problems with Mermaid diagrams in Kedro whereby they stopped rendering for no real reason, so let's keep an eye on this. What we did was to keep the Mermaid in the file but comment it out and instead screengrab the image (before commenting it out). So it's available to update relatively easily but we aren't expecting it to render on the fly.

Reply:
Ah, that's good to know! We currently also use it for the dynamic data section, but currently, I haven't encountered any issues yet 👍 What I like about it is the interactivity as you can click directly to the relevant links, which wouldn't work in a screenshot